styles/style.css

```diff
@@ -0,0 +1,312 @@
+@import url(../startrly/reset.css);
+@import url('https://fonts.googleapis.com/css?family=IBM+Plex+Sans|Open+Sans');
+@import url('https://use.fontawesome.com/releases/v5.8.1/css/all.css');
+
+body {
+  font-family: 'IBM Plex Sans', sans-serif;
+  font-size: 16px;
+  color: #676a81;
+  background: rgb(225, 247, 242);
+  max-width:1920px;
+  margin:0 auto;
+  overflow-x:hidden;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  margin: 0;
+  color:#43485c;
+  font-size: 20px;
+  font-family: 'IBM Plex Sans', sans-serif;
```

Comment:
I'm not sure whether you need to redefine the font-family, since you already did that in the `body` rule-set.